taskcluster/ci/collect-mono-trg/kind.yml

```diff
@@ -37,7 +37,7 @@ task-defaults:
     worker-type: b-linux-large
     worker:
         docker-image: {"in-tree": "train"}
-        max-run-time: 86400
+        max-run-time: 2592000
```

Comment:
If you squint - you will realize this is not going to work 😅

fwiw it would be caught by a test run